data-reconciliation-app/Makefile

```diff
@@ -58,4 +58,12 @@ e2e-test: ## 📦 Run end to end tests
 
 unit-test: ## 🔨 Run the Application unit-test
 	@echo -e "\e[34m$@\e[0m" || true
-	@npm run unit-test
\ No newline at end of file
+	@npm run unit-test
+	
+# Run `make start-host` in another terminal before running this.
+# To build and start a ccf network and automatically deploy your application on it.
+demo-code: ## Demo
```

Comment:
Can this be just "demo" and then it aligns with the banking-app

Reply:
I can do that but then I have to rename the demo folder.
Should I rename the folder to demo-app?

Reply:
Renamed the folder.